src/app/base/components/SectionHeader/_index.scss

```diff
@@ -1,6 +1,11 @@
 @mixin SectionHeader {
   .section-header {
     padding-top: $spv--medium;
+    position: sticky;
+    top: 0;
+    background: $color-x-light;
+    border-bottom: $border;
+    z-index: 2; // Ensure title displays above notification bar when scrolled
```

Comment:
This change should land in `maas-react-components` as part of the [content section](https://canonical.github.io/maas-react-components/?path=/docs/sections-contentsection--docs#with-form ).

But first we should confirm with @dgtlntv if that's how we want this component to behave. Content section is also used across various forms and it's not clear if it should have a sticky header everywhere (or at all).

Reply:
That the header is sticky is nice imo. I can't say at the moment whether it's something that should be used everywhere, as I don't know in what other contexts the content area is used.

Reply:
@Jay-Topher Thanks. But please open a PR for maas-react-components with this change. I'd suggest the sticky header behaviour being applied through a prop passed onto the Header component. It should probably be disabled by default.